lib/k8s-e2e-tests/src/metrics.rs

```diff
@@ -9,7 +9,7 @@ pub async fn load(url: &str) -> Result<String, Box<dyn std::error::Error>> {
 
 fn metrics_regex() -> regex::Regex {
     regex::RegexBuilder::new(
-        r"^(?P<name>[a-zA-Z_:][a-zA-Z0-9_:]*)(?P<labels>\{[^}]*\})? (?P<value>.+)$",
+        r"^(?P<name>[a-zA-Z_:][a-zA-Z0-9_:]*)(?P<labels>\{[^}]*\})? (?P<value>[^ ]+?)( (?P<timestamp>[^ ]+?))?$",
```

Comment:
I don't think the `value` is optional as above, and `\S` is a nice short-hand for `[^ ]`:
```suggestion
        r"^(?P<name>[a-zA-Z_:][a-zA-Z0-9_:]*)(?P<labels>\{[^}]*\})? (?P<value>\S+)( (?P<timestamp>[^ ]+?))?$",
```

Reply:
The `?` here is a lazy qualifier for `+`, not an optional qualifier for a group.
I didn't know about `\S`, thx!

Reply:
Added `\S`.